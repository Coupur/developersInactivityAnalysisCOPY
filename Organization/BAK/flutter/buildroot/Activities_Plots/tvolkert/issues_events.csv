id,date,event,creator_login
1116113413,2017-06-08 19:22:04,review_requested,tvolkert
1116113606,2017-06-08 19:22:13,review_requested,tvolkert
1116130275,2017-06-08 19:33:39,referenced,tvolkert
1116130276,2017-06-08 19:33:39,merged,tvolkert
1116130277,2017-06-08 19:33:39,closed,tvolkert
1116130331,2017-06-08 19:33:42,head_ref_deleted,tvolkert
