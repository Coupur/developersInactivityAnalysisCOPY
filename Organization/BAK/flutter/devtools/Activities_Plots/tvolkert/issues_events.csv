id,date,event,creator_login
2313225576,2019-05-01 23:44:57,transferred,tvolkert
