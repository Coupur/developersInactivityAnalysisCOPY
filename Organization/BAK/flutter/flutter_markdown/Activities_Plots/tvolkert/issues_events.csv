id,date,event,creator_login
1272496933,2017-09-29 22:47:28,referenced,tvolkert
1272496934,2017-09-29 22:47:28,merged,tvolkert
1272496938,2017-09-29 22:47:28,closed,tvolkert
1272496967,2017-09-29 22:47:30,head_ref_deleted,tvolkert
1272396242,2017-09-29 21:16:14,review_requested,tvolkert
1272396430,2017-09-29 21:16:23,review_requested,tvolkert
1272411930,2017-09-29 21:27:41,referenced,tvolkert
1272411932,2017-09-29 21:27:41,merged,tvolkert
1272411934,2017-09-29 21:27:41,closed,tvolkert
1272411993,2017-09-29 21:27:43,head_ref_deleted,tvolkert
