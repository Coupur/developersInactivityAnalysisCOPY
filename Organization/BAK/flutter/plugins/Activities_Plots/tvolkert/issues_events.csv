id,date,event,creator_login
2056517243,2019-01-06 23:46:54,mentioned,tvolkert
2056517244,2019-01-06 23:46:54,subscribed,tvolkert
1981270644,2018-11-21 23:03:11,review_requested,tvolkert
1761948943,2018-07-31 07:09:09,mentioned,tvolkert
1761948945,2018-07-31 07:09:09,subscribed,tvolkert
1500170470,2018-03-01 23:21:28,mentioned,tvolkert
1500170471,2018-03-01 23:21:28,subscribed,tvolkert
1502286737,2018-03-03 01:21:44,referenced,tvolkert
1502286738,2018-03-03 01:21:45,merged,tvolkert
1502286739,2018-03-03 01:21:45,closed,tvolkert
1501347492,2018-03-02 15:16:53,mentioned,tvolkert
1501347495,2018-03-02 15:16:53,subscribed,tvolkert
1368961115,2017-12-02 18:10:01,referenced,tvolkert
1368961116,2017-12-02 18:10:01,merged,tvolkert
1368961118,2017-12-02 18:10:01,closed,tvolkert
1368961134,2017-12-02 18:10:04,head_ref_deleted,tvolkert
1270501823,2017-09-28 19:56:10,review_requested,tvolkert
1271958999,2017-09-29 16:22:51,referenced,tvolkert
1271959001,2017-09-29 16:22:51,merged,tvolkert
1271959004,2017-09-29 16:22:51,closed,tvolkert
1271959094,2017-09-29 16:22:54,head_ref_deleted,tvolkert
1232229093,2017-09-02 04:20:52,mentioned,tvolkert
1232229094,2017-09-02 04:20:52,subscribed,tvolkert
1169284947,2017-07-18 22:53:38,referenced,tvolkert
1169284949,2017-07-18 22:53:38,merged,tvolkert
1169284953,2017-07-18 22:53:39,closed,tvolkert
1163536426,2017-07-14 00:15:08,review_requested,tvolkert
1163559107,2017-07-14 00:46:36,referenced,tvolkert
1163559108,2017-07-14 00:46:36,merged,tvolkert
1163559109,2017-07-14 00:46:36,closed,tvolkert
1163559140,2017-07-14 00:46:38,head_ref_deleted,tvolkert
1146764425,2017-06-30 23:33:40,review_requested,tvolkert
1151334635,2017-07-05 17:21:54,referenced,tvolkert
1151334637,2017-07-05 17:21:54,merged,tvolkert
1151334638,2017-07-05 17:21:54,closed,tvolkert
1151334761,2017-07-05 17:21:58,head_ref_deleted,tvolkert
1145008840,2017-06-29 21:14:59,review_requested,tvolkert
1145146237,2017-06-29 23:15:26,referenced,tvolkert
1145146238,2017-06-29 23:15:26,merged,tvolkert
1145146240,2017-06-29 23:15:26,closed,tvolkert
1145146280,2017-06-29 23:15:28,head_ref_deleted,tvolkert
1143400191,2017-06-29 01:13:43,review_requested,tvolkert
1144762034,2017-06-29 18:33:38,head_ref_force_pushed,tvolkert
1144866558,2017-06-29 19:40:46,referenced,tvolkert
1144866559,2017-06-29 19:40:46,merged,tvolkert
1144866564,2017-06-29 19:40:47,closed,tvolkert
1144866635,2017-06-29 19:40:50,head_ref_deleted,tvolkert
1140712857,2017-06-27 15:11:20,review_requested,tvolkert
1140902357,2017-06-27 16:52:48,referenced,tvolkert
1140902360,2017-06-27 16:52:48,merged,tvolkert
1140902362,2017-06-27 16:52:48,closed,tvolkert
1140902410,2017-06-27 16:52:50,head_ref_deleted,tvolkert
1140713043,2017-06-27 15:11:25,review_requested,tvolkert
1140847595,2017-06-27 16:20:49,referenced,tvolkert
1140847597,2017-06-27 16:20:49,merged,tvolkert
1140847602,2017-06-27 16:20:49,closed,tvolkert
1140847710,2017-06-27 16:20:53,head_ref_deleted,tvolkert
1137571827,2017-06-24 21:27:50,review_requested,tvolkert
1137571828,2017-06-24 21:27:50,review_requested,tvolkert
1137601607,2017-06-25 00:28:00,head_ref_force_pushed,tvolkert
1139484863,2017-06-26 21:41:27,referenced,tvolkert
1139484865,2017-06-26 21:41:27,merged,tvolkert
1139484866,2017-06-26 21:41:27,closed,tvolkert
1139484988,2017-06-26 21:41:33,head_ref_deleted,tvolkert
1142320565,2017-06-28 13:04:33,mentioned,tvolkert
1142320566,2017-06-28 13:04:33,subscribed,tvolkert
1143223481,2017-06-28 21:54:12,mentioned,tvolkert
1143223482,2017-06-28 21:54:12,subscribed,tvolkert
1133830847,2017-06-21 23:07:20,mentioned,tvolkert
1133830848,2017-06-21 23:07:20,subscribed,tvolkert
1135115669,2017-06-22 16:48:41,mentioned,tvolkert
1135115670,2017-06-22 16:48:41,subscribed,tvolkert
1130132604,2017-06-20 00:47:42,review_requested,tvolkert
1130132605,2017-06-20 00:47:42,review_requested,tvolkert
1130194838,2017-06-20 02:10:52,referenced,tvolkert
1130194839,2017-06-20 02:10:52,merged,tvolkert
1130194840,2017-06-20 02:10:52,closed,tvolkert
1130194873,2017-06-20 02:10:55,head_ref_deleted,tvolkert
1127703243,2017-06-17 00:48:29,closed,tvolkert
1127495549,2017-06-16 20:55:02,review_requested,tvolkert
1127611960,2017-06-16 22:33:14,referenced,tvolkert
1127611961,2017-06-16 22:33:14,merged,tvolkert
1127611962,2017-06-16 22:33:14,closed,tvolkert
1127611987,2017-06-16 22:33:16,head_ref_deleted,tvolkert
1123748398,2017-06-14 17:40:41,mentioned,tvolkert
1123748399,2017-06-14 17:40:41,subscribed,tvolkert
1121577550,2017-06-13 15:10:45,review_requested,tvolkert
1121652083,2017-06-13 15:45:53,referenced,tvolkert
1121652085,2017-06-13 15:45:53,merged,tvolkert
1121652088,2017-06-13 15:45:53,closed,tvolkert
1121652186,2017-06-13 15:45:57,head_ref_deleted,tvolkert
1120733414,2017-06-13 06:16:44,review_requested,tvolkert
1120733415,2017-06-13 06:16:44,review_requested,tvolkert
1120733417,2017-06-13 06:16:44,review_requested,tvolkert
1120799919,2017-06-13 07:16:17,head_ref_force_pushed,tvolkert
1121671768,2017-06-13 15:56:08,head_ref_force_pushed,tvolkert
1121745472,2017-06-13 16:38:02,referenced,tvolkert
1121745473,2017-06-13 16:38:02,merged,tvolkert
1121745474,2017-06-13 16:38:02,closed,tvolkert
1121745522,2017-06-13 16:38:04,head_ref_deleted,tvolkert
1118448421,2017-06-11 07:25:59,review_requested,tvolkert
1118448500,2017-06-11 07:26:21,review_requested,tvolkert
1118824013,2017-06-12 04:58:13,head_ref_force_pushed,tvolkert
1118830454,2017-06-12 05:11:21,head_ref_force_pushed,tvolkert
1118881346,2017-06-12 06:28:08,head_ref_force_pushed,tvolkert
1119673255,2017-06-12 15:15:50,head_ref_force_pushed,tvolkert
1119832789,2017-06-12 16:42:02,head_ref_force_pushed,tvolkert
1120114590,2017-06-12 19:28:03,referenced,tvolkert
1120114592,2017-06-12 19:28:03,merged,tvolkert
1120114602,2017-06-12 19:28:03,closed,tvolkert
1120114830,2017-06-12 19:28:12,head_ref_deleted,tvolkert
1114193772,2017-06-07 18:24:16,review_requested,tvolkert
1114193972,2017-06-07 18:24:23,review_requested,tvolkert
1114304656,2017-06-07 19:35:35,referenced,tvolkert
1114304657,2017-06-07 19:35:35,merged,tvolkert
1114304658,2017-06-07 19:35:35,closed,tvolkert
1114304732,2017-06-07 19:35:38,head_ref_deleted,tvolkert
1110846330,2017-06-05 23:06:12,review_requested,tvolkert
1110884032,2017-06-05 23:49:31,referenced,tvolkert
1110884034,2017-06-05 23:49:31,merged,tvolkert
1110884037,2017-06-05 23:49:31,closed,tvolkert
1110884057,2017-06-05 23:49:34,head_ref_deleted,tvolkert
1110741845,2017-06-05 21:36:10,mentioned,tvolkert
1110741847,2017-06-05 21:36:10,subscribed,tvolkert
1110314526,2017-06-05 16:57:40,review_requested,tvolkert
1110363582,2017-06-05 17:29:39,referenced,tvolkert
1110363585,2017-06-05 17:29:39,merged,tvolkert
1110363596,2017-06-05 17:29:40,closed,tvolkert
1110363803,2017-06-05 17:29:49,head_ref_deleted,tvolkert
1110844742,2017-06-05 23:04:23,referenced,tvolkert
1110846125,2017-06-05 23:05:54,referenced,tvolkert
1110884039,2017-06-05 23:49:32,referenced,tvolkert
1088537408,2017-05-18 20:51:38,review_requested,tvolkert
1088537409,2017-05-18 20:51:38,review_requested,tvolkert
1088549727,2017-05-18 21:00:01,review_requested,tvolkert
1090169728,2017-05-19 20:29:09,closed,tvolkert
1090169854,2017-05-19 20:29:13,head_ref_deleted,tvolkert
1086912320,2017-05-17 23:20:20,closed,tvolkert
