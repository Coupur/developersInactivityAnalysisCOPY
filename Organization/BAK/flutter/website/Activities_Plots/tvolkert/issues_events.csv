id,date,event,creator_login
2459135208,2019-07-04 00:41:35,mentioned,tvolkert
2459135209,2019-07-04 00:41:35,subscribed,tvolkert
2454758188,2019-07-02 15:21:13,mentioned,tvolkert
2454758190,2019-07-02 15:21:13,subscribed,tvolkert
2409346654,2019-06-13 04:40:44,mentioned,tvolkert
2409346655,2019-06-13 04:40:44,subscribed,tvolkert
2368162423,2019-05-27 02:17:33,labeled,tvolkert
2373027808,2019-05-28 20:22:23,mentioned,tvolkert
2373027809,2019-05-28 20:22:23,subscribed,tvolkert
2374891957,2019-05-29 12:14:49,mentioned,tvolkert
2374891959,2019-05-29 12:14:49,subscribed,tvolkert
2310016402,2019-04-30 16:19:56,referenced,tvolkert
2310016404,2019-04-30 16:19:56,merged,tvolkert
2310016406,2019-04-30 16:19:56,closed,tvolkert
2310016507,2019-04-30 16:19:58,head_ref_deleted,tvolkert
2260908687,2019-04-08 20:27:31,mentioned,tvolkert
2260908688,2019-04-08 20:27:31,subscribed,tvolkert
2260399483,2019-04-08 17:12:30,mentioned,tvolkert
2260399485,2019-04-08 17:12:30,subscribed,tvolkert
2327612467,2019-05-08 15:26:05,labeled,tvolkert
2327612468,2019-05-08 15:26:05,labeled,tvolkert
2327612471,2019-05-08 15:26:05,labeled,tvolkert
2016763341,2018-12-10 18:57:27,mentioned,tvolkert
2016763345,2018-12-10 18:57:27,subscribed,tvolkert
2005247391,2018-12-04 19:26:28,mentioned,tvolkert
2005247392,2018-12-04 19:26:28,subscribed,tvolkert
1946898070,2018-11-05 17:42:43,review_requested,tvolkert
1948167598,2018-11-06 07:14:11,mentioned,tvolkert
1948167599,2018-11-06 07:14:11,subscribed,tvolkert
1768903347,2018-08-03 00:01:43,mentioned,tvolkert
1768903348,2018-08-03 00:01:43,subscribed,tvolkert
1664917501,2018-06-05 23:10:49,mentioned,tvolkert
1664917502,2018-06-05 23:10:49,subscribed,tvolkert
1652372116,2018-05-29 23:53:17,mentioned,tvolkert
1652372118,2018-05-29 23:53:17,subscribed,tvolkert
1564651845,2018-04-09 19:50:45,mentioned,tvolkert
1564651847,2018-04-09 19:50:45,subscribed,tvolkert
1565302855,2018-04-10 05:07:33,mentioned,tvolkert
1565302856,2018-04-10 05:07:33,subscribed,tvolkert
1447791359,2018-01-30 04:58:33,mentioned,tvolkert
1447791360,2018-01-30 04:58:33,subscribed,tvolkert
1448949216,2018-01-30 16:34:24,mentioned,tvolkert
1448949218,2018-01-30 16:34:24,subscribed,tvolkert
1134175919,2017-06-22 06:55:08,mentioned,tvolkert
1134175920,2017-06-22 06:55:08,subscribed,tvolkert
1055654514,2017-04-24 22:14:00,referenced,tvolkert
1055654515,2017-04-24 22:14:00,merged,tvolkert
1055654518,2017-04-24 22:14:00,closed,tvolkert
1055654542,2017-04-24 22:14:01,head_ref_deleted,tvolkert
921108200,2017-01-12 17:47:42,referenced,tvolkert
921108202,2017-01-12 17:47:42,merged,tvolkert
921108203,2017-01-12 17:47:42,closed,tvolkert
