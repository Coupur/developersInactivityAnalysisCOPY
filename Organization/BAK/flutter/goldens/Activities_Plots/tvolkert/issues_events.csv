id,date,event,creator_login
1950231722,2018-11-06 22:09:12,referenced,tvolkert
1950231724,2018-11-06 22:09:12,merged,tvolkert
1950231727,2018-11-06 22:09:12,closed,tvolkert
1946802938,2018-11-05 17:04:24,referenced,tvolkert
1946802941,2018-11-05 17:04:24,merged,tvolkert
1946802947,2018-11-05 17:04:24,closed,tvolkert
